CHANGELOG.md

```diff
@@ -1,3 +1,4 @@
-# x.x.x (yyyy-mm-dd)
+# 4.0.0b0 (2021-04-30)
+
+- Created Action Hub template component. [#1](https://github.com/blackbaud/skyux-pages/pull/1)
```

Comment:
```suggestion
- Created the action hub component. [#1](https://github.com/blackbaud/skyux-pages/pull/1)
```
Based on the docs at https://github.com/blackbaud/skyux-pages/pull/4, I assume we don't need `template` as part of the component name. Am I interpreting that correctly?